lib/gpu/gpuCommand.hpp

```diff
@@ -22,6 +23,8 @@ class gpuDeviceInterface;
 /// Enumeration of known GPU command types.
 enum class gpuCommandType { COPY_IN, BARRIER, KERNEL, COPY_OUT, NOT_SET };
 
+class gpuCommandState : public kotekan::kotekanLogging {};
```

Comment:
Might be good to have a comment on what this class is used for here at the base.

Reply:
Done!